src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate(operation, value) {
+      this.result = operation(value, this.result);
```

Comment:
pass just value from operate

```suggestion
    operate(operation, value) {
      operation(value);
```